docs/source/usersguide/input.rst

```diff
@@ -1116,6 +1116,15 @@ Each ``material`` element can have the following attributes or sub-elements:
     .. note:: If one nuclide is specified in atom percent, all others must also
               be given in atom percent. The same applies for weight percentages.
 
+    An optional attribute/sub-element for each nuclide is ``scattering``. This
+    attribute may be set to "ace" to use the scattering laws specified in the
+    ACE files (default). Alternatively, when set to "lab", the ACE scattering
```

Comment:
I agree that "lab" is not a very intuitive descriptor. I also think that "isotropic" is not particularly intuitive since it may be with respect to either the lab or CM frames (and most users might assume CM). I just pushed a commit which uses "iso-in-lab" instead. What do you think of this?